### PR TITLE
Remove Margins added to page with SCCS and Map after case search

### PR DESCRIPTION
**Pull request by Jtang-1**
## Product Description

This change keeps split screen case search a full width. Previously, searching again after already being already showing `CaseTileListView` would remove the full width.

Existing behavior shown below:

https://github.com/dimagi/commcare-hq/assets/88759246/9463ec8d-ef30-403e-8dba-859040c9fa37

New shown below:

https://github.com/dimagi/commcare-hq/assets/88759246/3c9e9635-3418-4b80-b160-1bbd1f73e62f

## Technical Summary

[USH-4155 
](https://dimagi.atlassian.net/browse/USH-4155?focusedCommentId=308907&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-308907)

## Feature Flag

[Split Screen Case Search](https://www.commcarehq.org/hq/flags/edit/split_screen_case_search/)

### Safety story

locally tested. Change does not affect data and only visual.

### Automated test coverage

no automated tests exist

### QA Plan

no QA

### Rollback instructions

- [X] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [X] Risk label is set correctly
- [X] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js [hunk 1]
```diff
@@ -1161,10 +1161,6 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             }
         },
 
-        onBeforeDetach: function () {
-            CaseTileListView.__super__.onBeforeDetach.apply(this, arguments);
```
Comment (MartinRiese) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js:1165]: Is detaching the small screen listener not important?

Comment (Jtang-1) [reply corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js:1165]: `CaseTileListView` extends `CaseListView` so `onBeforeDetach` defined in `CaseListView` will still apply

**Comment by nospame** [comment]
Does this cause any issues with the main web apps container remaining full width if you, say, navigate via breadcrumbs back to the home screen? I think that's why this cleanup was added, though clearly it wasn't exactly right.

**Comment by Jtang-1** [comment]
@nospame I added a screen recording of some navigation with this change implemented.  On any navigation, the logic [here](https://github.com/dimagi/commcare-hq/blob/067b3ef7ca8ed4026c62ea0dcee26ddfa459cc95/corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js#L113-L117) would either add the `full-width` if sidebar is enabled, or clean it up if not enabled. That should cover all the navigation cases

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js (hunk 1 above)
Comment (MartinRiese) [reply corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js:1165]: makes sense

**Review by nospame — APPROVED** [review]

**Comment by nospame** [comment]
> @nospame I added a screen recording of some navigation with this change implemented. On any navigation, the logic [here](https://github.com/dimagi/commcare-hq/blob/067b3ef7ca8ed4026c62ea0dcee26ddfa459cc95/corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js#L113-L117) would either add the `full-width` if sidebar is enabled, or clean it up if not enabled. That should cover all the navigation cases

Not seeing that screen recording (I'm probably not looking in the right place) but I tested locally for my own reassurance and it looks good.

**Comment by Jtang-1** [comment]
@nospame Oops i guess i forgot to save it. Just uploaded now in the description. Thanks for checking!

**Pull request merged**